server/src/main/java/com/datastrato/gravitino/server/web/rest/ExceptionHandlers.java

```diff
@@ -181,13 +177,23 @@ public Response handle(OperationType op, String object, String parent, Exception
 
       String errorMsg =
           String.format(
-              BASE_MSG_TEMPLATE,
-              formattedObject,
-              op.name(),
-              formattedParent,
-              e.getClass().getSimpleName());
+              BASE_MSG_TEMPLATE, formattedObject, op.name(), formattedParent, getErrorMsg(e));
       LOG.error(errorMsg, e);
       return Utils.internalError(errorMsg, e);
     }
+
+    protected String getErrorMsg(Throwable throwable) {
+      if (throwable == null || throwable.getMessage() == null) {
+        return "";
+      }
+
+      String message = throwable.getMessage();
+      int pos = message.lastIndexOf(": ");
```

Comment:
I'm afraid the pattern ": " is not unique to the error message, for example, if the error message reads "Failed to load schema: (schema content...)".

Reply:
This is a real-life example:
<img width="1771" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/2b5ce3fe-b2be-4b34-9b9b-9fe20f96dfd2">

Reply:
> This is a real-life example: <img alt="image" width="1771" src="https://private-user-images.githubusercontent.com/15794564/298481915-2b5ce3fe-b2be-4b34-9b9b-9fe20f96dfd2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5MjIzODQsIm5iZiI6MTcwNTkyMjA4NCwicGF0aCI6Ii8xNTc5NDU2NC8yOTg0ODE5MTUtMmI1Y2UzZmUtYjJiZS00YjM0LTliOWItOWZlMjBmOTZkZmQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIyVDExMTQ0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBkZDI3ZDk4OWI2MGMzY2IzNjNlMzRiZjU2YTc5MGRmNWI3OWUxMjc4MjcwYTdlMzFhODBhNWY5Nzc5NDQ5YmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qxiDMkvEX9pTxyYlo7Bi9o3i1r7xuv9MpVQ-4ctY8Qk">

Can you paste the full exception log? I only parse from the outer exception, not the inner one.

Reply:
A simple way is to use "exception.getMessage()", but the message like "java.lang.RuntimeException: java.lang.IllegalArgumentException:" is too verbose, what I wanted is to try to avoid this. Do you have any better solution?

Reply:
Maybe you can use "Exception: " to replace ": " if all Exception class ends with `Exception`.

Reply:
Yeah, but we can always have an exceptional case. 😢

Reply:
Agreed. I have no better idea yet.